### PR TITLE
feat(scripts/install.sh): don't pass install token via command line

### PR DESCRIPTION
**Pull request by swiatekm-sumo**
I'm leaving the actual option in, as it's currently still displayed in the UI, but I'm removing it from documentation and manifests for Chef, Puppet and Ansible.

### CODE REVIEW

**Review by aiwaniuk — APPROVED** [review]
Thanks

**Comment by swiatekm-sumo** [comment]
Modified the tests as well.

File: examples/chef/sumologic-otel-collector/resources/default.rb [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
 
 def get_install_script_command(resource)
-  command_parts = ["bash", INSTALL_SCRIPT_PATH, "--installation-token #{resource.install_token}"]
+  command_parts = ["SUMOLOGIC_INSTALL_TOKEN=#{resource.install_token}", "bash", INSTALL_SCRIPT_PATH]
```
Comment (sumo-drosiek) [inline examples/chef/sumologic-otel-collector/resources/default.rb:61]: is it working if we pass `ENV` in chef as normal command?

**Review by sumo-drosiek — APPROVED** [review]
LGTM, one chef question

File: examples/chef/sumologic-otel-collector/resources/default.rb (hunk 1 above)
Comment (swiatekm-sumo) [reply examples/chef/sumologic-otel-collector/resources/default.rb:61]: That's supposed to work, but I replaced it with a more idiomatic approach from Chef's docs.

**Pull request merged**